scholarx/archive/index.html

```diff
@@ -210,5 +253,6 @@ <h6>UNIVERSITY</h6>
 <script src="/assets/js/navigation.js"></script>
 <!-- External JS file for the JS functions-->
 <script src="functions.js"></script>  
+
```

Comment:
We don't need this line right? Remove it @janithlahirukariyawasam

Reply:
Yes we can remove that line also

Reply:
Hmm, we have to upload there images right?

Reply:
```suggestion
```

Reply:
Commit suggestion @janithlahirukariyawasam